tests/integration/awslambda/test_lambda_api.py

```diff
@@ -2347,6 +2785,16 @@ def test_create_multiple_lambda_permissions(
 
 class TestLambdaUrl:
     @pytest.mark.skipif(condition=is_old_provider(), reason="not supported")
+    @pytest.mark.skip_snapshot_verify(
+        paths=[
+            # broken at AWS yielding InternalFailure
```

Comment:
If these are AWS issues, it would be better to use a transformer to ignore them instead of skipping the verification :thinking:

Reply:
that would need to be a transformer that makes the test succeed now (given the specific InternalFailure) but fails if AWS changes the behavior in a way that doesn't match our current implementation (which yields the conceptually correct error message). Hence, it should transform the specific `InternalFailure` into the conceptually expected exception, right 🤔

Reply:
replaced with transformer now
As discussed in Slack, it is somewhat suboptimal because it won't catch future changes but we should reserve `skip_snapshot_verify` for LS parity issues (i.e., TODOs). These are probably only a few edge cases and hence not worth investing in "reverse-parity" mismatch infrastructure.